iOSLearningApp/NavigationBar.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  NavigationBar.swift
+//  iOSLearningApp
+//
+//  Created by Chioma Amanda Mmegwa  on 11/10/2022.
+//
+
+import SwiftUI
+
+struct NavigationBar: View {
+    var navigationTitle: String
+    var rightIcon1: String?
+    var rightIcon2: String?
```

Comment:
could be PRivates?

Reply:
If I made them privates, I wont be able to use my custom NavigationBar objects outside this file.